src/groupByKey.ts

```diff
@@ -2,6 +2,17 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey<T>(items: T[], key: keyof T): GroupsMap<T> {
+  return items.reduce((grouped, item) => {
+    const keyValue = item[key] as string;
+    const newGrouped = grouped;
```

Comment:
why do we need this var if we can use `grouped` directly

Reply:
because lint blocks it

Reply:
![image](https://github.com/mate-academy/ts_group_by_key/assets/122530482/7a5fe4a6-b5f4-474a-b655-9eeae7cce7a3)

Reply:
i have used '--no-verify' for commit new changes